src/review/serializers/peer_review_serializer.py

```diff
@@ -24,6 +27,31 @@ class Meta:
             "updated_date",
         )
 
+    def get_user(self, obj):
```

Comment:
I'm pretty sure we don't need to do the following aggressive check just to get the author profile image.

Without doing the checks below, I'm getting the following error from S3

![image](https://github.com/user-attachments/assets/968fe7c3-1827-434c-b484-9bff753b27ed)

Links generated look like this: https://researchhub-staging-storage.s3.amazonaws.com/uploads/author_profile_images/2024/07/27/blob?AWSAccessKeyId=AKIAXYKJSD5I3RHQXF5G&Signature=Oyya7bob6qv8HlGiyAmTymieEVo%3D&Expires=1732153442